contrib/docker/Dockerfile.dev

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:latest
```

Comment:
That is surprising that the ubuntu 16.04 image is smaller than Jessie. There is a ```debian:jessie-slim``` image that is around 80 mb but we need to test it out to ensure it works as expected.